images/demo/README.md

```diff
@@ -5,22 +5,22 @@ This demo guides you through the basics of using `kamu` and its key concepts. It
 ## Requirements
 To run this demo you'll only need:
 * `docker`
-* `docker-compose`
 
 ## Running
 First you will need to download the `docker-compose.yml` file:
 
-```bash
+```shell
 wget https://raw.githubusercontent.com/kamu-data/kamu-cli/master/images/demo/docker-compose.yml
 ```
 
 To run all demo components on your computer simply do:
 
-```bash
-docker-compose up
+```shell
+docker compose up
 ```
 
-> If you've run the environment before you might want to get latest versions of the images using `docker-compose pull` command.
+> [!TIP]
+> If you've run the environment before you might want to get latest versions of the images using `docker compose pull` command.
```

Comment:
> [!TIP]

Allows us to style the blockquote, see more [here](https://github.com/orgs/community/discussions/16925)